parser/tests/Create.test

```diff
@@ -139,6 +139,62 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+# Empty partitioning attribute is not allowed
```

Comment:
Thanks @zuyu! Can you please provide more details about the output file? Like where will it be generated and what will it be named as?

I only see the following when the test fails.

[gerald@francisco] (91)$ make test
Running tests...
Test project /scratch/gerald/quickstep/build/parser
      Start  1: quickstep_parser_tests_ParserTest_aggregate
 1/11 Test  #1: quickstep_parser_tests_ParserTest_aggregate ...   Passed    0.01 sec
      Start  2: quickstep_parser_tests_ParserTest_alter
 2/11 Test  #2: quickstep_parser_tests_ParserTest_alter .......   Passed    0.01 sec
      Start  3: quickstep_parser_tests_ParserTest_copy
 3/11 Test  #3: quickstep_parser_tests_ParserTest_copy ........   Passed    0.01 sec
      Start  4: quickstep_parser_tests_ParserTest_create
 4/11 Test  #4: quickstep_parser_tests_ParserTest_create ......***Failed    0.01 sec
      Start  5: quickstep_parser_tests_ParserTest_delete
 5/11 Test  #5: quickstep_parser_tests_ParserTest_delete ......   Passed    0.01 sec
      Start  6: quickstep_parser_tests_ParserTest_drop
 6/11 Test  #6: quickstep_parser_tests_ParserTest_drop ........   Passed    0.01 sec
      Start  7: quickstep_parser_tests_ParserTest_index
 7/11 Test  #7: quickstep_parser_tests_ParserTest_index .......   Passed    0.01 sec
      Start  8: quickstep_parser_tests_ParserTest_insert
 8/11 Test  #8: quickstep_parser_tests_ParserTest_insert ......   Passed    0.01 sec
      Start  9: quickstep_parser_tests_ParserTest_select
 9/11 Test  #9: quickstep_parser_tests_ParserTest_select ......   Passed    0.01 sec
      Start 10: quickstep_parser_tests_ParserTest_tpch
10/11 Test #10: quickstep_parser_tests_ParserTest_tpch ........   Passed    0.01 sec
      Start 11: quickstep_parser_tests_ParserTest_update
11/11 Test #11: quickstep_parser_tests_ParserTest_update ......   Passed    0.01 sec

91% tests passed, 1 tests failed out of 11

Total Test time (real) =   0.08 sec

The following tests FAILED:
          4 - quickstep_parser_tests_ParserTest_create (Failed)
Errors while running CTest
make: *** [test] Error 8